wpk/run.sh

```diff
@@ -85,6 +86,10 @@ main() {
                     HAVE_PKG_NAME_WIN=true
                 elif [ "${PKG_NAME: -4}" == ".pkg" ]; then
                     HAVE_PKG_NAME_MAC=true
+                elif [ "${PKG_NAME: -4}" == ".rpm" ]; then
```

Comment:
What about APK?

Reply:
Added in ed90d7125ea40a260c7764955ea6dbdba6f1a759